vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -155,6 +162,17 @@ def parse_pod(pod)
         end
       end
 
+      createdby_txt = pod.metadata.annotations.try("kubernetes.io/created-by")
```

Comment:
wow I didn't know Ruby was *that* flexible...but it is really odd.  Is that because method_missing is being leveraged in kubeclient?  You may want a comment clarifying that, yes, you are indeed calling a method named that, so the next person to read this code doesn't go all o_O  

cc @tenderlove @jrafanie Thought you might like to see this one :smile: